addons/l10n_id_efaktur_coretax/__manifest__.py

```diff
@@ -0,0 +1,38 @@
+# -*- encoding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Indonesia E-faktur (Coretax)',
+    'icon': '/l10n_id/static/description/icon.png',
+    'version': '1.0',
+    'description': """
+        E-invoicing feature provided by DJP (Indonesian Tax Office). As of January 1st 2025,
+        Indonesia is using CoreTax system, which changes the file format and content of E-Faktur.
+        We're changing from CSV files into XML.
+        At the same time, due to tax regulation changes back and forth, for general E-Faktur now,
+        TaxBase (DPP) has to be mulitplied by factor of 11/12 while multiplied to tax of 12% which
+        is resulting to 11%.
+    """,
+    'category': 'Accounting/Localizations/EDI',
+    'depends': ['l10n_id', 'l10n_id_efaktur'],
```

Comment:
Efaktur being auto install, I think this one should too. New databases shouldn't have the efaktur module installed alone.